packages/manager/src/features/CloudPulse/shared/CloudPulseResourcesSelect.tsx

```diff
@@ -132,10 +132,12 @@ export const CloudPulseResourcesSelect = React.memo(
         autoHighlight
         clearOnBlur
         data-testid="resource-select"
-        disabled={disabled || isLoading}
+        disabled={disabled}
+        errorText={isError ? `Failed to fetch ${label || 'Resources'}` : ''}
```

Comment:
Same here, if label is undefined?

Reply:
@kmuddapo , it is handled, please see a || condition
we will show **Failed to fetch Resources**